ndcube/utils/wcs.py

```diff
@@ -32,8 +32,20 @@ def inv(self):
 # types and International Virtual Observatory Alliance vocabulary.
 # See http://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html
 wcs_to_ivoa = {
+    "DSUN": "custom:pos.distance.sunToObserver",
+    "HGLT": "custom:pos.heliographic.stonyhurst.lon",
+    "CRLT": "custom:pos.heliographic.carrington.lat",
+    "HGLN": "custom:pos.heliographic.stonyhurst.lat",
+    "CRLN": "custom:pos.heliographic.carrington.lon",
+    "HECR": "custom:pos.distance.fromSunCenter",
+    "HECR": "custom:pos.distance.fromSunSurface",
+    "SOLX": "custom.pos.distance.heliocentric.west",
+    "SOLY": "custom.pos.distance.heliocentric.north",
+    "SOLZ": "custom.pos.distance.heliocentric.sunToObserver",
```

Comment:
Should this be ```custom.pos.distance.heliocentric.alt```?

Same question for ```"HPRZ"```.

Reply:
In fact, ```"SOLX"```, ```"SOLY"```, ```"SOLZ"``` should probably be ```"custom:pos.heliocentric.x"```, ```"custom:pos.heliocentric.y"```, ```"custom:pos.heliocentric.z"```.

Reply:
I *think* we can drop the custom for these?

Reply:
In the [list of allowed words](https://www.ivoa.net/documents/REC/UCD/UCDlist-20070402.html#_Toc163276347) I only see ```pos.heliocentric```, not ```pos.heliocentric.x/y/z```.  Whereas I do see ```pos.cartesian.x```.